librz/core/analysis_objc.c

```diff
@@ -78,15 +78,15 @@ static ut64 readQword(RzCoreObjc *objc, ut64 addr, bool *success) {
 }
 
 static void objc_analyze(RzCore *core) {
-	const char *oldstr = rz_core_notify_begin(core, "Analyzing code to find selref references");
+	rz_core_notify_begin(core, "Analyzing code to find selref references");
```

Comment:
What about `const char *msg = "Analyzing code to find selfref references"` and using it instead of duplicating the string?